Uplift/Assets.xcassets/share-light.imageset/Contents.json

```diff
@@ -2,15 +2,17 @@
   "images" : [
     {
       "idiom" : "universal",
-      "filename" : "madeline.jpg",
+      "filename" : " Dark.png",
```

Comment:
? "madeline.jpg" changed to "Dark.png"?

Reply:
I think because I deleted all of the Pro profile images (of which `madeline.jpg` was one) and added the new icon images git though I just changed the name of one image.